conda_libmamba_solver/solver.py

```diff
@@ -18,6 +18,7 @@
 
 import libmambapy as api
 from conda import __version__ as _conda_version
+from conda._vendor.boltons.setutils import IndexedSet
```

Comment:
I wish we would have boltons in defaults :-/

Reply:
Honestly we can probably get away with a manually de-duped `tuple` here as well, but since we are using `IndexedSet` in more places already, I guess it doesn't hurt. It feels weird to import from `_vendor` though.

Reply:
Yeah, I think it would be good to start the process to package boltons in defaults to make this possible, we use IndexSet elsewhere in conda, but it's silly that just bundle a subset of boltons with it. Not a problem of this library. Unless you want to wait for the process of boltons to land?

Reply:
Scratch that, I've requested the package at Anaconda, so we may be able to switch to it in a future release. Filed https://github.com/conda/conda-libmamba-solver/issues/112

Reply:
The package is available on defaults, if you're interested to make this part of this PR?